backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/process/ModCntr_Specific_Price_Update_Selection.java

```diff
@@ -197,7 +197,7 @@ public ImmutableSet<ModCntrSpecificPriceId> retrieveContractSpecificPricesFromSe
 					.subQuery(modules)
 					.end();
 
-			if (modularContractSettingsService.isMatchingComputingMethodType(p_ModCntr_Type_ID, ComputingMethodType.AverageAddedValueOnShippedQuantity))
+			if (modularContractSettingsService.isMatchingComputingMethodType(p_ModCntr_Type_ID, ComputingMethodType.PurchaseAverageAddedValueOnShippedQuantity))
```

Comment:
Shouldn't this be similar for SalesAverageAddedValueOnShippedQuantity ?

Reply:
good catch, thx